deploy_scripts/deploy_local.sh

```
@@ -0,0 +1,38 @@
+
+set -eu
+
+docker_compose() {
+  #export DOCKER_HOST="ssh://ytdlor.local"
+  export COMPOSE_PROJECT_NAME="ytdlor-production"
+  export SECRET_KEY_BASE="13a1d85b904da9a62716e15705cf814eb9dddfa82df51075da9b0e3cfbb51a9ff9a70b23d0e6e33157896a038a2327fd826e6e509d1d208c912f852339d563e7"
+  export YTDLOR_WEB_PORT="8001"
+  docker compose -f docker-compose.yml -f docker-compose-production.yml "${@}"
+}
+
+args="${@:-""}"
```

Shellcheck

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)